ironfish-cli/src/commands/wallet/notes.ts

```diff
@@ -34,19 +36,8 @@ export class NotesCommand extends IronfishCommand {
 
     for await (const note of response.contentStream()) {
       CliUx.ux.table(
-        [note],
+        [{ ...note, assetName: BufferUtils.toHuman(Buffer.from(note.assetName, 'hex')) }],
```

Comment:
Won't this already be done in tablecols Asset?

Reply:
Updated!